### PR TITLE
Use python exe

### PR DESCRIPTION
**Pull request by Garulf**
Need to specify file is run with Python

### CODE REVIEW

**Comment by Garulf** [comment]
Odd it's not skipping Python test when `plugins.json` is *not* changed

**Comment by Garulf** [comment]
Ah, validator has the same typo which i copied and pasted...

**Comment by Garulf** [comment]
Successfully skip if plugins.json isn't changed 🤞

File: .github/workflows/test-python.yml [hunk 1]
```diff
@@ -5,16 +5,18 @@ jobs:
...
-        with:
-          skip_after_successful_duplicate: 'true'
-          paths: '["plugins.json"]'
+    - uses: actions/checkout@v2
+    - uses: dorny/paths-filter@v2
+      id: filter
+      with:
+        filters: |
+          plugins:
+            - 'plugins.json'
```
Comment (taooceros) [inline .github/workflows/test-python.yml:16]: Why change to this?

Comment (Garulf) [reply .github/workflows/test-python.yml:16]: Not sure exactly why but the old workflow kept detecting changes to plugins.json regardless if it was changed in PR or not.

**Comment by jjw24** [comment]
awesome, thanks for this. will test this in the other pr before merging in.

**Comment by Garulf** [comment]
It's not breaking anything but validator is probably having the same issue.

**Review by jjw24 — APPROVED** [review]

**Pull request merged**